dashboard/app/controllers/lessons_controller.rb

```diff
@@ -27,6 +27,20 @@ def show
     @lesson_data = @lesson.summarize_for_lesson_show(@current_user)
   end
 
+  # GET /s/script-name/lessons/1/student
+  def student_lesson_plan
+    script = Script.get_from_cache(params[:script_id])
+    return render :forbidden unless script.is_migrated && script.include_student_lesson_plans
+
+    @lesson = script.lessons.find do |l|
+      l.has_lesson_plan && l.relative_position == params[:lesson_position].to_i
+    end
+    raise ActiveRecord::RecordNotFound unless @lesson
+
+    @lesson_data = @lesson.summarize_for_student_lesson_plan
+    render 'lessons/student_lesson_plan'
```

Comment:
this line should be unnecessary -- the rails convention is to render this view by default.